### PR TITLE
WIP Add interactive contact form

### LINKED ISSUE #8256: [Smart start] modal contact us form
The modal contact us form should be built from a copy doc here: https://docs.google.com/document/d/1DobeTFjupLbGl4TUMu4f4pPAqNMYrR6U4U303ioYkV4/edit?usp=sharing

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Interactive/Dynamic form on /smart-start

## QA

- Go to the demo, https://ubuntu-com-8324.demos.haus/smart-start
- Scroll down to almost the bottom and click on the "Get in touch" link. (This is just a demo, still needs a better location to show this form e.g. landing index page?)

## Issue / Card

Fixes #8256

### CODE REVIEW

File: templates/smart-start/document.html [hunk 1]
```diff
@@ -9,5 +9,8 @@
         {{ document.body_html | safe }}
     </div>
 </div>
+<a href="/support/contact-us" class="js-invoke-modal">Get in touch&nbsp;›</a>
```
Comment (anthonydillon) [inline templates/smart-start/document.html:12]: Please use `&rsaquo;` instead of >

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Code looks good with one comment

File: templates/smart-start/document.html (hunk 1 above)
Comment (carkod) [reply templates/smart-start/document.html]: This is just an example of this functionality. Galem is making a landing page, which I suppose is where I am supposed to add this link/button? I will hold off this PR until that is ready.

**Comment by SirSamTumless** [comment]
Waiting on the briefing and completion of the /smart-start page.

**Comment by carkod** [comment]
Replaced by https://github.com/canonical-web-and-design/ubuntu.com/pull/8452

**Pull request closed without merging**